[ODS-4299] Cleanup remaining bulk references

semalaiappan opened the pull request.

blmeyers commented on Application/EdFi.Ods.WebService.Tests/App.config:
```diff
@@ -20,10 +20,7 @@
-    <add key="QueuePrefix" value="EdFiOds" />
-    <add key="QueueAutoCreate" value="1" />
-    <add key="CommitUploadCommandMessageEndPoint" value="localhost" />
+    <add key="SqlServerDataDirectory" value="C:\Program Files\Microsoft SQL Server\MSSQL10_50.MSSQLSERVER\MSSQL\DATA" />        
```
This one is also for bulk

semalaiappan replied:
yes removed "SqlServerDataDirectory"

blmeyers commented on Application/EdFi.Ods.WebService.Tests/App.config:
```diff
@@ -20,10 +20,7 @@
     <!-- Uncomment WebDriver setting to see UI tests run in Firefox-->
     <!--<add key="WebDriver" value="Firefox"/>-->
     <!-- Sql Server 2008 by Default-->
```
Also for bulk

semalaiappan replied:
yes removed 3 lines

blmeyers commented on Application/EdFi.Ods.WebService.Tests/App.config:
```diff
@@ -20,10 +20,7 @@
-    <add key="QueueAutoCreate" value="1" />
-    <add key="CommitUploadCommandMessageEndPoint" value="localhost" />
+    <add key="SqlServerDataDirectory" value="C:\Program Files\Microsoft SQL Server\MSSQL10_50.MSSQLSERVER\MSSQL\DATA" />        
     <add key="StartOperationCommandMessageEndpoint" value="localhost" />
```
Also for bulk

semalaiappan replied:
yes removed "StartOperationCommandMessageEndpoint"

blmeyers commented on Application/EdFi.Ods.WebService.Tests/App.config:
```diff
@@ -20,10 +20,7 @@
+    <add key="SqlServerDataDirectory" value="C:\Program Files\Microsoft SQL Server\MSSQL10_50.MSSQLSERVER\MSSQL\DATA" />        
     <add key="StartOperationCommandMessageEndpoint" value="localhost" />
     <add key="PreserveAllTestSdks" value="true" />
     <add key="ReleaseBulkDatabaseAfterProcessing" value="true" />
```
Also for bulk

semalaiappan replied:
yes removed "ReleaseBulkDatabaseAfterProcessing"

semalaiappan commented:
I removed .gitignore file: #File System Queues section
I removed Application\EdFi.Ods.WebService.Tests\WebApiInstaller.cs: line 99, commented out queue installer
